.travis.yml

```diff
@@ -14,10 +14,11 @@ services:
 
 env:
   matrix:
-    - TEST_SUITE=standard NODE_VERSION=8 USE_COUCHDB=false USE_YARN=false
-    - TEST_SUITE=normal NODE_VERSION=8 USE_COUCHDB=true USE_YARN=true
-    - TEST_SUITE=normal NODE_VERSION=9 USE_COUCHDB=true USE_YARN=true
-    - TEST_SUITE=performance NODE_VERSION=8 USE_COUCHDB=true USE_YARN=true
+    - TEST_SUITE=standard NODE_VERSION=8 USE_COUCHDB=false
+    - TEST_SUITE=normal NODE_VERSION=8 USE_COUCHDB=true
+    - TEST_SUITE=normal NODE_VERSION=9 USE_COUCHDB=true
```

Comment:
I don't think we need to test v9 as we only use LTS (even numbers) in production. Let's drop this out of the matrix to reduce build times.